.taskcluster.yml

```diff
@@ -1,232 +1,321 @@
 # yamllint disable rule:line-length
 # This file is rendered via JSON-e by
-# - hg-push - https://hg.mozilla.org/ci/ci-admin/file/default/build-decision/
+# - github events - https://github.com/taskcluster/taskcluster/tree/main/services/github
 # - cron tasks - https://hg.mozilla.org/ci/ci-admin/file/default/build-decision/
 # - action tasks - taskcluster/taskgraph/actions/registry.py
 ---
 version: 1
+reporting: checks-v1
+policy:
+    pullRequests: collaborators
```

Comment:
Yeah, I'm actually going to be deleting that secret since this PR is deleting the webhook task :)

I had pretty much only set it to collaborators because I figured it was safer to do that for now and make it public later.. Maybe once I delete the secret I'll submit another PR to change it to public.